package.json

```diff
@@ -31,7 +31,9 @@
     "doc:generate": "node_modules/jsdoc/jsdoc.js src/vl-*.js README.md --destination doc",
     "release:prepare": "echo tjiek tsjak vollen bak",
     "release:testless": "np --yolo",
-    "demo": "http-server"
+    "demo": "http-server",
+    "test:server:start": "http-server &",
```

Comment:
Misschien `test:server:start` vervangen door `pretest` zodat je in `test:e2e` het `test:server:start` script niet manueel moet aanroepen?

Reply:
Ik zal dit later in een ander ticket doen want dit komt nu overeen met de run scripts uit alle andere componenten.